lib/matplotlib/backends/web_backend/js/mpl.js

```diff
@@ -253,60 +282,48 @@ mpl.figure.prototype._init_toolbar = function () {
             // put a spacer in here.
             continue;
         }
-        var button = $('<button/>');
-        button.addClass(
-            'ui-button ui-widget ui-state-default ui-corner-all ' +
-                'ui-button-icon-only'
-        );
-        button.attr('role', 'button');
-        button.attr('aria-disabled', 'false');
-        button.click(method_name, toolbar_event);
-        button.mouseover(tooltip, toolbar_mouse_event);
-
-        var icon_img = $('<span/>');
-        icon_img.addClass('ui-button-icon-primary ui-icon');
-        icon_img.addClass(image);
-        icon_img.addClass('ui-corner-all');
-
-        var tooltip_span = $('<span/>');
-        tooltip_span.addClass('ui-button-text');
-        tooltip_span.html(tooltip);
-
-        button.append(icon_img);
-        button.append(tooltip_span);
-
-        nav_element.append(button);
+        var button = document.createElement('button');
+        button.classList =
+            'ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only';
+        button.setAttribute('role', 'button');
+        button.setAttribute('aria-disabled', 'false');
+        button.addEventListener('click', on_click_closure(method_name));
+        button.addEventListener('mouseover', on_mouseover_closure(tooltip));
+
+        var icon_img = document.createElement('span');
+        icon_img.classList =
+            'ui-button-icon-primary ui-icon ' + image + ' ui-corner-all';
+
+        var tooltip_span = document.createElement('span');
+        tooltip_span.classList = 'ui-button-text';
+        tooltip_span.innerHTML = tooltip;
+
+        button.appendChild(icon_img);
+        button.appendChild(tooltip_span);
+
+        toolbar.appendChild(button);
     }
 
-    var fmt_picker_span = $('<span/>');
+    var fmt_picker_span = document.createElement('span');
 
-    var fmt_picker = $('<select/>');
-    fmt_picker.addClass('mpl-toolbar-option ui-widget ui-widget-content');
-    fmt_picker_span.append(fmt_picker);
-    nav_element.append(fmt_picker_span);
-    this.format_dropdown = fmt_picker[0];
+    var fmt_picker = document.createElement('select');
+    fmt_picker.classList = 'mpl-toolbar-option ui-widget ui-widget-content';
+    fmt_picker_span.appendChild(fmt_picker);
+    toolbar.appendChild(fmt_picker_span);
+    this.format_dropdown = fmt_picker;
 
     for (var ind in mpl.extensions) {
         var fmt = mpl.extensions[ind];
-        var option = $('<option/>', {
-            selected: fmt === mpl.default_extension,
-        }).html(fmt);
-        fmt_picker.append(option);
+        var option = document.createElement('option');
+        option.selected = fmt === mpl.default_extension;
+        option.innerHTML = fmt;
+        fmt_picker.appendChild(option);
     }
 
-    // Add hover states to the ui-buttons
-    $('.ui-button').hover(
```

Comment:
It looks like this is gone?  Do we care or did it get moved someplace else?

Reply:
It didn't seem to do anything; we can do hover with CSS now.

Reply:
it seems to still work so without this so I guess we are not worried.

Reply:
sorry, I had not refreshed an did not see your comment before I responded again :sheep: